opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py

```diff
@@ -125,6 +132,9 @@ def export(
     def shutdown(self, timeout_millis: float = 30_000, **kwargs) -> None:
         pass
 
+    def force_flush(self, timeout_millis: float = 10_000) -> bool:
```

Comment:
Is there no implementation logic for `force_flush`?

Reply:
Yes, There is nothing pending to flush here. This exists only for spec compliance and completing the exporter interface.